.gitignore

```diff
@@ -41,3 +41,10 @@ Thumbs.db
 
 # Next.js
 .next
+
+# dev database
+db/gddy.sqlite
```

Comment:
Removed the sqlite database from source control as it will likely be a source of conflict in the project going forward. I added a new NPM script to run the create/seed script and am calling that in `setup` now.